linkup-cli/Cargo.toml

```diff
@@ -40,6 +41,8 @@ sha2 = "0.10.8"
 hex = "0.4.3"
 tar = "0.4.43"
 flate2 = "1.0.35"
+# NOTE(augustoccesar)[2025-02-19]: This old version is to match the one being used by cloudflare
+mockito = "0.31"
```

Comment:
Shouldn't this be dev dependency?

Reply:
It should! Changed on 13b541753de16bdb6d40a39845eb018d21ecc90b